src/index.js

```diff
@@ -115,6 +125,33 @@ class DropzoneController extends Controller {
     }
   }
 
+  createTiffThumbnail(file, callback) {
+    const self = this;
```

Comment:
Why do you need this? Shouldn't the method below already be bound to this properly?